lang/en/abilities.php

```diff
@@ -39,6 +39,12 @@
         'tabs'  => [
             'abilities' => 'Sub abilities',
             'entities'  => 'Entities',
+            'reorder'   => 'Reorder Abilities',
```

Comment:
You can move all of these translations to `entities/abilities.php` instead, since it's related to entity abilities, and note the ability objects directly.